src/app/games/coerceo/coerceo.component.html

```diff
@@ -18,11 +18,10 @@
         </g>
     </g>
     <g *ngFor="let line of board; index as y">
-        <g *ngFor="let spaceContent of line; index as x"
-           id="click_{{ x }}_{{ y }}"
-           (click)="onClick(x, y)">
+        <g *ngFor="let spaceContent of line; index as x">
             <polygon *ngIf="isPyramid(x, y)"
```

Comment:
coordsAndContents then !

Reply:
Oui mais là au moins ce serait monoligné !
Aussi: conflicte pas mon ticket malotru !

Reply:
ah ouais, j'aurais de quoi résoudre comme conflit

Reply:
en vrai avec les merges, c'est presque au niveau ou pour être 100% sûr faudrait tester toute la colonne "to test in develop" en une fois avant l'enmasterisation, et cette période de test ne devant souffrir aucun ajout de code pour être supeeeeeer sûr... Mais bon c'pas si sensible que ça :D !

Reply:
yep, idéalement faut faire un spot check de dev avant master, mais si nos tests sont bons on est bon !